sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1606 @@
+{
```

Comment:
<p>Clear all cell outputs for all cells in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>We have intentionally kept outputs in the notebook, as per feedback we got from our science team. The notebooks take several tens of minutes to run. This is too much time for a customer who just wants to see what the product does. We plan to do this for all clarify notebooks later. Internal AWS account IDs in the outputs have been replace with dummy id.</p>